### PR TITLE
feat: started onboard weaviate

### PR DESCRIPTION
**Pull request by Walkoss**
## Summary
- Added weaviate application https://github.com/pluralsh/plural-artifacts/issues/629

## Checklist

- [x]  No images hosted from dockerhub
- [ ]  Are dashboards present to understand the health of the application.  There **must** be at least 1 of these
    - [ ]  all databases should have dashboards
    - [ ]  ideally also have at least cpu/mem utilization dashboards for webserver tier of the app
    - [ ]  you can use `plural from-grafana` to convert a grafana dashboard found via google to our CRD
- [ ]  Are scaling runbooks present
    - [ ]  all databases **must** have scaling runbooks
    - [ ]  you can use the charts in `pluralsh/module-library` to accelerate this
- [ ]  do you need to add config overlays?
    - [ ]  inputing secrets
    - [ ]  configuring autoscaling
- [ ]  If there’s a web-facing component to the app, we need to support OIDC authentication and setting up private networks if no authentication option is viable
- [x]  All major clouds must be supported
    - [x]  Azure
    - [x]  AWS
    - [x]  GCP

### CODE REVIEW

**Review by michaeljguarino — COMMENTED** [review]
we still need to merge #631 to get the contributor tags to reflect, but once that's in we can get this available

File: weaviate/helm/weaviate/values.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    kubernetes.io/tls-acme: "true"
+    cert-manager.io/cluster-issuer: letsencrypt-prod
+  service:
+    port: 80
+
+weaviate:
+  image:
+    registry: dkr.plural.sh
+    tag: 1.18.0
+    repo: weaviate/semitechnologies/weaviate
```
Comment (Walkoss) [inline weaviate/helm/weaviate/values.yaml:30]: @michaeljguarino can you confirm that all images in `vendor_images.yaml` will be sync within `dkr.plural.sh` registry with application name as prefix?

**Comment by Walkoss** [comment]
> we still need to merge #631 to get the contributor tags to reflect, but once that's in we can get this available

@michaeljguarino I can remove those for now in order to continue this new application integration with features like backup in bucket, authenticiation with OIDC and so on, wdyt?

File: weaviate/helm/weaviate/values.yaml (hunk 1 above)
Comment (michaeljguarino) [reply weaviate/helm/weaviate/values.yaml:30]: that's correct

**Comment by michaeljguarino** [comment]
> > we still need to merge #631 to get the contributor tags to reflect, but once that's in we can get this available
> 
> @michaeljguarino I can remove those for now in order to continue this new application integration with features like backup in bucket, authenticiation with OIDC and so on, wdyt?

Do you want to do that this weekend?  I can theoretically unblock if you want to work on it quickly otherwise could pick it up on Monday as well

**Comment by Walkoss** [comment]
> > > we still need to merge #631 to get the contributor tags to reflect, but once that's in we can get this available
> > 
> > 
> > @michaeljguarino I can remove those for now in order to continue this new application integration with features like backup in bucket, authenticiation with OIDC and so on, wdyt?
> 
> Do you want to do that this weekend? I can theoretically unblock if you want to work on it quickly otherwise could pick it up on Monday as well

Yes, I would like to work on it today. As it is private, will I be able to bundle install it with the CLI? If not, that's not a problem, I will helm install and terraform apply manually. 
Otherwise I can publish this app on my own plural account and then remove it later to integrate it here? I didn't try publishing on my own account but could be interesting to see. Could I install it with the plural cli (to check the configuration, etc)?

**Comment by michaeljguarino** [comment]
You can technically publish it via your own account but we'd have to manually reparent it under the standard plural publisher if we were to bring it back in this repository fully, which is ok.  Let me try to get you unblocked though

**Review by michaeljguarino — APPROVED** [review]

**Comment by Walkoss** [comment]
> You can technically publish it via your own acco

Thank you

**Pull request merged**